packages/clerk-js/src/ui/components/UserButton/useMultisessionActions.tsx

```diff
@@ -27,8 +27,11 @@ export const useMultisessionActions = (opts: UseMultisessionActionsParams) => {
     if (otherSessions.length === 0) {
       return signOut(opts.navigateAfterSignOut);
     }
-    return signOut(opts.navigateAfterMultiSessionSingleSignOut, { sessionId: session.id }).finally(() =>
-      card.setIdle(),
+    return signOut(opts.navigateAfterMultiSessionSingleSignOut, { sessionId: session.id }).finally(
+      () =>
+        void navigate(`${opts.signInUrl}/choose`).finally(() => {
+          card.setIdle();
+        }),
```

Comment:
@alexcarpenter Wouldn't this redirect twice ? Once when `navigateAfterMultiSessionSingleSignOut` is called and then to `/choose` ?

I think this logic should go inside `navigateAfterMultiSessionSingleSignOut`

Reply:
@panteliselef did a bit of pairing with @dstaley which can be seen here https://github.com/clerk/javascript/pull/4203/commits/0026ca54189d15f0c4fbab913a90f55cb342796a

Let me know what you think. Definitely being cautious welcomed 😄 added a few tests to validate the updates.